File: launchdarkly/typescript-axios/src/DeleteIntegrationConfigurationExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.IntegrationsBetaApi(configuration).deleteIntegrationConfiguration(
  "integrationConfigurationId_string", // integrationConfigurationId
).catch((error: Error | AxiosError) => {
  console.log("Exception when calling IntegrationsBetaApi#deleteIntegrationConfiguration:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});

File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/DeleteIntegrationConfigurationExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class DeleteIntegrationConfigurationExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey.Add("Authorization", "YOUR_API_KEY");

        try
        {
            new IntegrationsBetaApi(config).DeleteIntegrationConfiguration(
                integrationConfigurationId: "integrationConfigurationId_string"
            );
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling IntegrationsBetaApi#DeleteIntegrationConfiguration: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
